src/EditorFeatures/TestUtilities/SignatureHelp/AbstractSignatureHelpProviderTests.cs

```diff
@@ -347,31 +339,30 @@ private string ToString(IEnumerable<TaggedText> list)
 
         protected async Task VerifyItemWithReferenceWorkerAsync(string xmlString, IEnumerable<SignatureHelpTestItem> expectedOrderedItems, bool hideAdvancedMembers)
         {
-            using (var testWorkspace = TestWorkspace.Create(xmlString))
-            {
-                var cursorPosition = testWorkspace.Documents.First(d => d.Name == "SourceDocument").CursorPosition.Value;
-                var documentId = testWorkspace.Documents.First(d => d.Name == "SourceDocument").Id;
-                var document = testWorkspace.CurrentSolution.GetDocument(documentId);
-                var code = (await document.GetTextAsync()).ToString();
+            using var testWorkspace = TestWorkspace.Create(xmlString);
```

Comment:
disable whitespace in the preview to see that nothing changed here (except moving to simple-using statemnets).